src/containers/Masthead/components/MastheadSearch.jsx

```diff
@@ -86,7 +86,7 @@ const MastheadSearch = ({
       search: `?${queryString.stringify({
         query: query.length > 0 ? query : undefined,
         subjects,
-        levels: filterIds,
+        levels: filterIds || undefined,
```

Comment:
For å unngå at søkesida får tom streng som levels-parameter. Dette sender undefined dersom filterIds er tom streng.